src/services/refactors/moveToNewFile.ts

```diff
@@ -954,6 +978,24 @@ function isExported(sourceFile: SourceFile, decl: TopLevelDeclarationStatement,
         getNamesToExportInCommonJS(decl).some(name => sourceFile.symbol.exports!.has(escapeLeadingUnderscores(name)));
 }
 
+function getDefaultExportName(sourceFile: SourceFile) {
+    return tryCast((sourceFile.symbol.exports?.get("default" as __String)?.declarations?.[0] as undefined | ExportAssignment)?.expression, isIdentifier);
+}
+
+function isExportedLaterInOldFile(sourceFile: SourceFile, decl: TopLevelDeclarationStatement, useEs6Exports: boolean, name: Identifier, toMove: readonly Statement[]): boolean {
+    if (useEs6Exports) {
+        return !isExpressionStatement(decl)
+            && !hasSyntacticModifier(decl, ModifierFlags.Export)
+            && (!!sourceFile.symbol.exports?.has(name.escapedText) || getDefaultExportName(sourceFile)?.escapedText === name.escapedText)
+            && !toMove.some((node) =>
+                (isExportDeclaration(node) && tryCast(node.exportClause, isNamedExports)?.elements.some(el => el.name.text === name.text))
+                || (isExportAssignment(node) && tryCast(node.expression, isIdentifier)?.text === name.text));
+    }
+    return false;
+    // return !!sourceFile.symbol && !!sourceFile.symbol.exports &&

```

Comment:
Forgot about this branch. And I just checked that cases from linked issue are also appliable for commonjs, so I see this branch also needs to be updated...

Reply:
Okay, I give up, I'd better to not touch cjs as I filled linked issue for esm

Reply:
What was the problem with non-esm exports/imports?

Reply:
```ts
[|const test = 5|]
exports.test = test
```

In new file export is not added (only `const test = 5`)

But, I didn't find a way to figure out when the exported should be added and when it shouldn't.
I completely don't understand why `test` on the first line and `test` on the second are completely different symbols.

Reply:
Can you add a version of the tests with the commonjs cases? Then I might be able to help with dealing with symbols in the commonjs case.

Reply:
Done: https://github.com/zardoy/typescript/blob/f4e3d5667b22c36b7a541ae3f3e17f8467ec6995/tests/cases/fourslash/moveToNewFile_exportedLaterCjs.ts#L25 But I see the point of not prioritizing cjs problems, as I see its quite not common case: specific refactoring action + cjs codebase